backend/getIP.php

```diff
@@ -55,6 +55,10 @@
     echo json_encode(['processedString' => $ip . " - link-local IPv4 access", 'rawIspInfo' => ""]);
     die();
 }
+if (preg_match('/^100\.([6-9][0-9]|1[0-2][0-7])\./', $ip) === 1) { // 100.64/10 CGNAT IPv4
```

Comment:
This regex doesn't quite work. It allows addresses such as `100.63.0.0` but blocks addresses like `100.118.0.0`. Perhaps something like `/^100\.(6[4-9]|[7-9]\d|1[0-1]\d|12[0-7])\./` would work better. Here is an example of some of the tests https://regex101.com/r/6WAdkr/1

```suggestion
if (preg_match('/^100\.(6[4-9]|[7-9]\d|1[0-1]\d|12[0-7])\./', $ip) === 1) { // 100.64/10 CGNAT IPv4
```